pro-planner-client/src/components/Poll.jsx

```diff
@@ -70,6 +91,12 @@ function Poll({poll, pollId}) {
                                 size='sm'
                                 onClick={handleVote}>
                             Vote
+                        </Button>{' '}
+                        <Button style={{marginTop: '10px'}}
+                                variant="danger"
```

Comment:
We have custom styles for certain types of buttons, you can use `variant="custom-danger"` here.